config/platforms-chromeos.yaml

```diff
@@ -103,6 +103,8 @@ platforms:
     base_name: jacuzzi
     mach: mediatek
     dtb: dtbs/mediatek/mt8183-kukui-jacuzzi-juniper-sku16.dtb
+    context:
+      test_character_delay: 10
```

Comment:
So this means this parameter is now added for all MTK chromebooks, right? In that case, can you please drop those lines for other ones? Setting it for jacuzzi means it will be part of `mediatek-chromebook-device`, and therefore picked up for other devices based on this definition.

Reply:
Done - thanks for the heads up!